src/sc/recipe/staticresources/bobtemplate/karma.conf.js

```diff
@@ -0,0 +1,28 @@
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['mocha', 'chai', 'sinon'],
+    files: [
+      './tests/*.spec.js'
+    ],
+    exclude: [],
+    preprocessors: {
+      './tests/*.spec.js': ['webpack']
+    },
+    // webpack configuration
+    webpack: require("./webpack.config.js"),
+    webpackMiddleware: {
+      stats: "errors-only"
+    },
+    reporters: ['progress'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
```

Comment:
Is it better to use Firefox instead? how is travis configuration?

Note that we can add multiple browsers here.

Reply:
It don't look trivial to change to Firefox (in a simple test); Let's keep Chrome